boards/common/particle-mesh/include/board.h

```diff
@@ -61,6 +61,44 @@ extern "C" {
 #define BTN0_MODE           GPIO_IN_PU
 /** @} */
 
+/**
+ * @name    Antenna selection configuration
+ * @{
+ */
+
+/** Choices in antenna outputs for the board's nRF radio
+ *
+ * @see board_nrfantenna_select */
+enum board_nrfantenna_selection {
+    /** The board's built-in antenna */
+    BOARD_NRFANTENNA_BUILTIN,
+    /** The board's uFL connector */
+    BOARD_NRFANTENNA_EXTERNAL,
+};
+
+/** Drive the on-board antenna switch to connect the nRF radio to a given @p
+ * choice of antenna output.
+ * */
+void board_nrfantenna_select(enum board_nrfantenna_selection choice);
+
+#ifdef BOARD_PARTICLE_XENON
+#define VCTL1_PIN           GPIO_PIN(0, 24)
+#define VCTL2_PIN           GPIO_PIN(0, 25)
+#endif
+
+#ifdef BOARD_PARTICLE_ARGON
+#define VCTL1_PIN           GPIO_PIN(0, 25)
+#define VCTL2_PIN           GPIO_PIN(0, 2)
+#endif
+
+#ifdef BOARD_PARTICLE_BORON
+#define VCTL1_PIN           GPIO_PIN(0, 7)
+#endif
+
+#define VCTLn_MODE          GPIO_OUT
```

Comment:
Is this needed?

Reply:
It seems to be customary to define that, it's used in the GPIO initialization, but if that's not a deliberate RIOT pattern but just something that's been sneaking in, I'm happy to remove it. (I'd prefer that.)

Reply:
I think this is a case of cargo cult 😉 
What other mode could that pin possibly have?